doc/kernel-limitations.md

```diff
@@ -195,7 +196,10 @@ files for read/write, such as `/proc/self/exe`. Currently Mystikos only implemen
 a subset of these virtual files that are commonly used. An application could
 fail if it depends on a virtual file that is not implemented yet.
 
-Some libc functions/symbols that are present in glibc but not in musl. Our CRT
-is based on musl. And we try to achieve glibc compatibility by adding those
-missing functions/symbols to musl. An application could fail to start if it
-references one of the functions/symbols that we haven't patched yet.
+Regarding C runtime (CRT), Mystikos' CRT is based on [musl](https://wiki.musl-libc.org/), some libc 
+functions/symbols that are present in glibc but not in musl. Although we try to 
+achieve glibc compatibility by adding those missing functions/symbols to musl, an
+application could fail to start if it references one of the functions/symbols
```

Comment:
In some cases application may start, but fail at runtime when referencing the missing symbol.